br/pkg/lightning/restore/table_restore.go

```diff
@@ -932,7 +934,14 @@ func (tr *TableRestore) importKV(
 			return errors.Trace(err)
 		}
 	}
-	err := closedEngine.Import(ctx, regionSplitSize)
+	if regionSplitKeys == 0 {
+		if regionSplitSize > int64(config.SplitRegionSize) {
+			regionSplitKeys = int64(float64(regionSplitSize) / float64(config.SplitRegionSize) * float64(config.SplitRegionKeys))
+		} else {
+			regionSplitKeys = int64(config.SplitRegionKeys)
+		}
+	}
```

Comment:
those code seems more like adjusting, maybe move into `config.Adjust()`

Reply:
These codes are adjusting. However it's a bit hard to put them in `config.Adjust()` because some extra instances are needed. Maybe left a TODO here and refactor this in the next PR?